env-defaults

```diff
@@ -218,6 +237,12 @@ BUILT_IN_TASK_PREFIX=
 # Version to use for all built-in tasks.  "latest" (local / sandbox), "sit", "uat", or "prod"
 BUILT_IN_TASK_VERSION=latest
 
+# AWS region where harmony is deployed, irrelevant for local development
+AWS_DEFAULT_REGION=us-west-2
+
+# The object store used for storing shapefiles. Only S3 is supported currently.
+OBJECT_STORE_TYPE=s3
```

Comment:
Good catch.  Thinking this should probably be used by defaultObjectStore.  Also thinking we may have over-engineered something to have an env var with only one valid value.